platform/ios/scripts/document.sh

```diff
@@ -32,6 +32,9 @@ mkdir -p ${OUTPUT}
 
 cp platform/ios/screenshot.png "${OUTPUT}"
 
+DEFAULT_THEME="platform/darwin/docs/theme"
+THEME=${JAZZY_THEME:-$DEFAULT_THEME}
```

Comment:
This script is usually run by a make rule, not by itself, so the `THEME` variable needs to be plumbed through to here via the Makefile. This PR will end up looking very similar to #7396. (Preferably, the macOS script and make rule are kept in sync with this script for predictability’s sake.)